libvmi/libvmi.h

```diff
@@ -796,6 +796,21 @@ const char* vmi_translate_v2sym(
     addr_t rva);
 
 /**
+ * Performs the translation from an RVA to a symbol for Linux with KASLR offset
```

Comment:
So this is not an RVA. This is an actual virtual address, so it should be called as such. We should also say this is not supported for Windows at the moment.